src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -52,10 +53,14 @@ public Command parseCommand(String userInput) throws ParseException {
         final String arguments = matcher.group("arguments");
         switch (commandWord.toLowerCase()) {
 
+        case AddPhotoCommand.COMMAND_WORD:
```

Comment:
You wanna add Command Alias?

Reply:
Yes if possible. Since alias would be ap then i'll make the prefix p